feature-libs/cart/_index.scss

```diff
@@ -1,12 +1,12 @@
 $useLatestStyles: true;
 
-@import '~@spartacus/styles/scss/core';
-@import '~@spartacus/styles/scss/cxbase/blocks/modal';
-@import '~@spartacus/styles/scss/misc/table';
+@import '@spartacus/styles/scss/core';
+@import '@spartacus/styles/scss/cxbase/blocks/modal';
+@import '@spartacus/styles/scss/misc/table';
 
-@import '~bootstrap/scss/functions';
-@import '~bootstrap/scss/variables';
-@import '~bootstrap/scss/_mixins';
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/_mixins';
```

Comment:
It seems like `~`  says to import from the `node_modules`  folder.
Can you think of a way that removing the `-` can change the import behaviour and break a custom app?

Reply:
https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules
https://github.com/angular/components/commit/f2ff9e31425f0e395e6926bcaf48f876688000d8